cuegui/cuegui/plugins/MonitorJobsPlugin.py

```diff
@@ -46,7 +47,8 @@
 PLUGIN_DESCRIPTION = "Monitors a list of jobs"
 PLUGIN_PROVIDES = "MonitorJobsDockWidget"
 REGEX_EMPTY_STRING = re.compile("^$")
-
+TIME_DELTA = 3
```

Comment:
This constant name is a bit generic IMO and it's generally preferred for the name to contain the time unit if it's a duration -- maybe something like `JOB_RESTORE_THRESHOLD_DAYS`?

Reply:
Sorry, I was suggesting to rename `TIME_DELTA` -> `JOB_RESTORE_THRESHOLD_DAYS`. I'm confused now what each of these do.

Based on the code it looks like these two lines should be:

```
# Maximum number of jobs that will be restored on CueGUI restart.
JOB_RESTORE_MAX_JOBS = 200
# Maximum age for a job to be restored on CueGUI restart.
JOB_RESTORE_THREADHOLD_DAYS = 3
```

Do I have that right?